CI.md

```diff
@@ -89,7 +89,9 @@ Those jobs often have matrix run strategy which runs several different variation
 
 Some of the jobs require variables stored as a [GitHub Secrets](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets)
 to perform operations on Google Cloud Platform. Currently these jobs are limited to Apache repository only.
+
 These variables are:
+ * `GCP_PYTHON_WHEELS_BUCKET` - Name of the bucket where python source distribution and wheels will be stored. Apache/Beam repository has it set to `beam-wheels-staging`.
```

Comment:
nit: Use `For example` as in the other change.

Reply:
Thanks. Changed as you suggested it.